blueprint/posts/comments.apib

```diff
@@ -1,7 +1,21 @@
 ## Comments [/comments/{id}]
 
+**Authorisation**
+
+Authenticated | Role      | GET | POST | PATCH | DELETE
+------------: | --------: | :-: | :--: | :---: | :----:
+No            | None      | Y   | N    | N     | N
+Yes           | None      | Y   | Y    | N*    | Y
```

Comment:
This should be `Y*` not `N*` as you *can* update the resource, just for a limited period.